app/src/main/res/values/strings.xml

```diff
@@ -1810,6 +1810,14 @@
     <!-- Text for the snackbar to confirm auto-close is enabled for inactive tabs -->
     <string name="inactive_tabs_auto_close_message_snackbar">Auto-close enabled</string>
 
+    <!-- Awesome bar suggestions' headers -->
+    <!-- Search suggestions title for Firefox Suggest. -->
+    <string name="firefox_suggest_header">Firefox Suggest</string>
```

Comment:
doesn't Android normally follow sentence casing for titles?

Reply:
Since this string is a move of a already existing string, I would advise not to move this since we know this needs to be uplift to v108.

Reply:
This is a product name.